test/e2e/persistentips_test.go

```diff
@@ -98,14 +102,60 @@ var _ = Describe("Persistent IPs", func() {
 
 			})
 
-			It("should garbage collect IPAMClaims after virtual machine deletion", func() {
+			It("should garbage collect IPAMClaims after VM deletion", func() {
 				Expect(testenv.Client.Delete(context.Background(), vm)).To(Succeed())
 				Eventually(testenv.IPAMClaimsFromNamespace(vm.Namespace)).
 					WithTimeout(time.Minute).
 					WithPolling(time.Second).
 					Should(BeEmpty())
 			})
 
+			It("should garbage collect IPAMClaims after VM foreground deletion", func() {
+				Expect(testenv.Client.Delete(context.Background(), vm, foregroundDeleteOptions())).To(Succeed())
+				Eventually(testenv.IPAMClaimsFromNamespace(vm.Namespace)).
+					WithTimeout(time.Minute).
+					WithPolling(time.Second).
+					Should(BeEmpty())
+			})
+
+			When("the VM is stopped", func() {
+				BeforeEach(func() {
+					By("Invoking virtctl stop")
+					output, err := exec.Command("virtctl", "stop", "-n", td.Namespace, vmi.Name).CombinedOutput()
+					Expect(err).NotTo(HaveOccurred(), output)
+
+					By("Ensuring VM is not running")
+					Eventually(testenv.ThisVMI(vmi), 360*time.Second, 1*time.Second).Should(
+						And(
+							Not(testenv.BeCreated()),
+							Not(testenv.BeReady()),
+						))
```

Comment:
Is more clear this way
```suggestion
						SatisfyAll(
							Not(testenv.BeCreated()),
							Not(testenv.BeReady()),
						))
```

Reply:
done